lib/Config.js

```diff
@@ -221,6 +221,17 @@ class Config {
             this.replicationGroupId = 'RG001';
         }
 
+        this.replicationEndpoints = [];
```

Comment:
One day not too far away we should consider using `joi` to validate S3 config structure and types, like is done in backbeat (thanks @rahulreddy for the idea), this would make validation much more concise and less error prone. But it's ok for now of course.